### DISCUSSION TITLE
gdm-settings on freebsd, Polkit problem

### DISCUSSION THREAD

**Discussion by nsthy** [realmazharhussain/gdm-settings#212] [Q&A]
there is no problem about compile and install,but when i run ,it says Polkit not found,but i have Polkit installed,i also check and modify the code ,bypass the error,but failed to apply any changes,so is there a chance to run properly on freebsd?

**Comment by realmazharhussain** [comment]
What does the following command print?

```sh
whereis pkexec
```

**Reply by nsthy** [reply]
pkexec: /usr/local/bin/pkexec

**Reply by realmazharhussain** [reply]
What do the following commands print (separately)?

```sh
pkexec --version; echo $?
```

```sh
gdm-settings -v
```

**Comment by nsthy** [comment]
1.
pkexec version 123
Segmentation fault
139

2.
gdm-settings is not avaliable in freebsd

gnome version is 42

**Reply by realmazharhussain** [reply]
> gdm-settings is not avaliable in freebsd

But you also said

> there is no problem about compile and install

So, I assumed you had gdm-settings installed on your system.

> pkexec version 123
Segmentation fault
139

Your Polkit installation seems to be broken.

Try

```sh
pkexec /usr/bin/true; echo $?
```

**Reply by nsthy** [reply]
run this pkexec /usr/bin/true; echo $?,i get a pop up window ask for password,and return 0

run gdm-settings -v,i get
INFO:gdms.gui: Application Version    = 4.1
INFO:gdms.gui: Operating System       = FreeBSD 14.0-RELEASE
INFO:gdms.gui: PackageType            = Unknown
INFO:gdms.gui: TEMP_DIR               = /home/nsthy/.cache/app/io.github.realmazharhussain.GdmSettings
INFO:gdms.gui: HOST_DATA_DIRS         = /usr/local/share:/usr/share
INFO:gdms.gui: ShellGresourceFile     = /usr/local/share/gnome-shell/gnome-shell-theme.gresource
INFO:gdms.gui: UbuntuGdmGresourceFile = None
INFO:gdms.gui: Default Gresource File = /usr/local/share/gnome-shell/gnome-shell-theme.gresource

still  Polkit not found

**Reply by nsthy** [reply]
also when i run meson setup,i get this,dont know if it matters
Program appstream-util found: NO

**Reply by realmazharhussain** [reply]
That seems to be a Polkit bug. Have you reached out to Polkit package maintainers for FreeBSD?
